thorlcr/thorutil/thbuf.cpp

```diff
@@ -1507,7 +1513,10 @@ class CSharedWriteAheadDisk : public CSharedWriteAheadBase
             mb.append((byte)0);
             size32_t len = mb.length();
             chunk.setown(getOutOffset(len)); // will find space for 'len', might be bigger if from free list
+            CCycleTimer startCycles;
             tempFileIO->write(chunk->offset, len, mb.toByteArray());
+            tempFileElapsedCycles += startCycles.elapsedCycles();
+            tempSizeSpilled += len;
```

Comment:
spillFileIO is already accumulating time+bytes, I think you can just return them from it in getStatistic